openshift_cli_installer/libs/clusters/ocp_cluster.py

```diff
@@ -139,6 +144,19 @@ def _add_s3_bucket_data(self):
             f"{self.cluster_info['name']}-{self.cluster_info['shortuuid']}.zip"
         )
 
+    def check_and_assign_aws_cluster_region(self):
+        if self.cluster_info["platform"] in [AWS_STR, AWS_OSD_STR]:
+            if not self.cluster_info.get("region"):
```

Comment:
check `if not self.cluster_info.get("region"):` first, no need to check platform if the user pass region.